docs/faq.md

```diff
@@ -228,40 +228,6 @@ With the release of version 5.3, we've updated how you can write your stories mo
 With the [`@storybook/components`](https://www.npmjs.com/package/@storybook/components) package, you get a set of icons that you can use to customize your UI. Use the table below as a reference while writing your addon or defining your Storybook global types.
 Go through this [story](https://5a375b97f4b14f0020b0cda3-wbeulgbetj.chromatic.com/?path=/story/basics-icon--labels) to see how the icons look.
 
-| accessibility  | accessibilityalt | add          | admin        | alert         |
```

Comment:
Thank you for removing this maintenance burden!

Reply:
You're welcome!